lib/app/features/feed/views/components/user_info/user_info.dart

```diff
@@ -36,8 +37,18 @@ class UserInfo extends HookConsumerWidget {
         }
         return ListItem.user(
           onTap: () => ProfileRoute(pubkey: pubkey).push<void>(context),
-          title: Text(userMetadataEntity.data.displayName),
-          subtitle: Text(prefixUsername(username: userMetadataEntity.data.name, context: context)),
+          title: Text(
+            userMetadataEntity.data.displayName,
+            style: TextStyle(
+              color: textColor,
+            ),
+          ),
+          subtitle: Text(
+            prefixUsername(username: userMetadataEntity.data.name, context: context),
+            style: TextStyle(
```

Comment:
i think we should  apply style with `context.theme.appTextThemes`

Reply:
Same here.